pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -197,6 +205,8 @@ message CommandSubscribe {
     repeated KeyValue metadata = 10;
 
     optional bool read_compacted = 11;
+
+	optional int64 schema_version = 12;
```

Comment:
When I subscribe, should I just pass the schema itself? How do I know the version that my current code for `MyClass.class` has?

Reply:
I have changed this to pass the schema itself.